apps/ledger-live-mobile/docs/llm_e2e_testing.md

```diff
@@ -81,7 +81,7 @@ Verify you have an emulator [installed](https://developer.android.com/studio/run
   - Debug: `pnpm mobile e2e:build -c android.emu.debug`
   - Release: `pnpm mobile e2e:build -c android.emu.release`
 - Run the tests
-  - Debug: First, run `pnpm mobile start` to run Metro bundler, then in a separate terminal window run `pnpm mobile e2e:test -c android.emu.debug`
+  - Debug: First, run `pnpm mobile start` to run Metro bundler, then in a separate terminal window run `pnpm mobile e2e:test -c android.emu.debug`. Be sure to set the variable `MOCK=1` in the app .env file.
```

Comment:
This bit I'm still not sure about, I'm going to try on my side and see if it's needed

Reply:
Could you change the comment to something like: `When developing locally, you may need to set the variable `MOCK=1` in the app .env file.`. I'm still not sure exactly why you needed this despite the detox build commands building with the `MOCK=1` variable already set.